packages/cra-template/template/src/index.js

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
+import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
 
-ReactDOM.render(
+const root = ReactDOM.createRoot(document.getElementById('root'));
```

Comment:
🥳🥳🥳🥳

Reply:
We don't currently use this template in any of our apps, but new use cases of our fork would. So it's a nice update!